libpromises/var_expressions.c

```diff
@@ -95,6 +96,13 @@ static size_t VarRefHash(const VarRef *ref)
     return (h & (INT_MAX - 1));
 }
 
+unsigned int VarRefHash_untyped(const void *ref,
+                                unsigned int seed ARG_UNUSED,
+                                unsigned int max)
+{
+    return VarRefHash(ref) & (max - 1);
+}
```

Comment:
Never mind, I misunderstood. As long as `max` is a power of two, this is valid. I think that deserves a comment here though (and perhaps in some of the other places where this pops up).